boltdesignsystem/bolt
# DS 614 Clean up patternlab quotes

## Jira

https://pegadigitalit.atlassian.net/browse/DS-614

## Summary

Updated the double-quotes to single-quotes on all examples of .twig files to the design system standards.

## Details

Updated files in:
 - `doc-site/src/pages/patternlab/_patterns/40-components`
 - `doc-site/src/pages/patternlab/_patterns/50-pages`

## How to test

Check if all necessary files were updated to the latest standards. Check if any other files should be updated.
